app/assets/javascripts/form_exit.js.erb

```diff
@@ -5,10 +5,20 @@
     if(outsideOfSteps) return;
 
     for(var i=0; i<links.length; i++){
-      var stepsLink = links[i].href.indexOf('steps') >= 0
-      var skipLink = links[i].href.indexOf('skip_') >= 0
+      var isAnExternalLink = true
+      var whiteList = [
+        '/privacy',
+        '/steps',
+        '/terms',
+        'skip_'
+      ]
+      for(var j=0; j<whiteList.length; j++){
+        if(links[i].href.indexOf(whiteList[j]) >= 0){
+          isAnExternalLink = false
+        }
+      }
 
-      if(!stepsLink && !skipLink){
+      if(isAnExternalLink){
         links[i].addEventListener('click', function(event){
```

Comment:
so for external links we want to warn them that the session is cleared? It doesn't look like that is what is happening in the linked gif (thanks for that, BTW!) but the way I am reading this, if an "external link" is clicked, we give a js alert. 

My thought was that we only wanted to show this warning if the link they click takes them to the root path?